Add PagerDuty Integration Docs (#1402)
getsentry/sentry-docs: src/collections/_documentation/workflow/integrations/global-integrations.md

These integrations will connect your Sentry to incident management platforms to increase your visibility into incidents.

### PagerDuty

> The globale PagerDuty integration allows you to connect your Sentry organization with one or more PagerDuty accounts, and start getting incidents triggered from Sentry alerts.

Reviewer: ```suggestion
The global PagerDuty integration allows you to connect your Sentry organization with one or more PagerDuty accounts, and start getting incidents triggered by Sentry alerts.
```

1. In Sentry, navigate to **Organization Settings** > **Integrations**

2. Find PagerDuty in the list of the available Global Integrations list and click **Install.**
    
    [{% asset pagerduty/pd1.png %}]({% asset pagerduty/pd1.png @path %})

3. Click **Add Installation.**

    [{% asset pagerduty/pd-3.png %}]({% asset pagerduty/pd-3.png @path %})

4. You'll then be redirected to sign into PagerDuty and choose the PD account you'd like to connect to the current Sentry organization you are in.

    [{% asset pagerduty/pd-4.png %}]({% asset pagerduty/pd-4.png @path %})

5. After picking the account you'll be prompted to add the PagerDuty services you want Sentry to have access to. Click **Connect** once you've added your services.  

    [{% asset pagerduty/pd-5.png %}]({% asset pagerduty/pd-5.png @path %})

> 6. The PagerDuty integration should now be installed, you'll be able to see the services connected by going to the **configure** page for your installation. 

Reviewer: ```suggestion
6. The PagerDuty integration should now be installed. You'll be able to see the services connected by going to the **configure** page for your installation. 
```

    [{% asset pagerduty/pd-6.png %}]({% asset pagerduty/pd-6.png @path %})

7. You can now set up rules to use the new integration under **Project Configuration** in the configure section of your installation. 

    [{% asset pagerduty/pd-7.png %}]({% asset pagerduty/pd-7.png @path %})

> If you re-generate an integration key for one of your services in PagerDuty you can manually update that value in configuration page for your PagerDuty installation. Additionally, you can add services by clicking the **Add Services** button in the top right. This will take you to same page as shown in step 5. 

Reviewer: ```suggestion
If you re-generate an integration key for one of your services in PagerDuty, you can manually update that value in the configuration page for your PagerDuty installation. Additionally, you can add services by clicking the **Add Services** button in the top right. This will take you to the same page, as shown in step 5. 
```

These integrations will connect your Sentry to incident management platforms to increase your visibility into incidents.

### PagerDuty
The global PagerDuty integration allows you to connect your Sentry organization with one or more PagerDuty accounts, and start getting incidents triggered from Sentry alerts.

1. In Sentry, navigate to **Organization Settings** > **Integrations**

2. Find PagerDuty in the list of the available Global Integrations list and click **Install.**
    
    [{% asset pagerduty/pd_not_installed.png %}]({% asset pagerduty/pd_not_installed.png @path %})

3. Click **Add Installation.**

    [{% asset pagerduty/pd_learn_more.png %}]({% asset pagerduty/pd_learn_more.png @path %})

> 4. You'll then be redirected to sign into PagerDuty and choose the PD account you'd like to connect to the current Sentry organization you are in.

Reviewer 2: Don't feel like we should use acronyms.
```suggestion
4. You'll then be redirected to sign into PagerDuty and choose the account you'd like to connect to the current Sentry organization you are in.
```

    [{% asset pagerduty/pd_account_picker.png %}]({% asset pagerduty/pd_account_picker.png @path %})

> 5. After picking the account you'll be prompted to add the PagerDuty services you want Sentry to have access to. Click **Connect** once you've added your services.  

Reviewer 2: ```suggestion
5. After picking the account, you'll be prompted to add the PagerDuty services you want Sentry to send incidents to. Click **Connect** once you've added your services.  
```
